Omise/Models/Models.cs

```diff
@@ -336,6 +336,8 @@ public partial class Charge : ModelBase {
         public string Currency { get; set; }
         [JsonProperty("description")]
         public string Description { get; set; }
+        [JsonProperty("expires_at")]
+        public string ExpiresAt { get; set; }
```

Comment:
datatype should be `DateTime` same as `paid_at`.

Reply:
Yeah it should be will change